src/styles/elements/_subscription-create.scss

```diff
@@ -72,6 +61,7 @@ $name: '' !default;
 
 .StripeElement {
   padding: 0.75em;
+  background-color: mercury.$color-white;
   border: 1px solid mercury.$color-grayLight;
   border-radius: 0.2em;
 }
```

Comment:
@dangodev just a heads up: mercury is a bit restrictive here. I can't apply input styles to stripe elements because it's an input inside an iframe inside a div instead of an actual input element.

(I can make PRs for some of these things later, but for now I'm just taking notes and focusing on getting all these styles up to date first)